scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -24,23 +21,62 @@ workflow ImportGenomes {
   }
 
   String docker_final = select_first([docker, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
-  Array[String] input_vcfs = read_lines(input_vcfs_list)
 
   call GetMaxTableId {
     input:
       sample_map = sample_map
   }
 
-  scatter (i in range(length(input_vcfs))) {
-    if (defined(input_metrics)) {
-      File input_metric = select_first([input_metrics])[i]
-    }
+  call CreateTables as CreateMetadataTables {
+  	input:
+      project_id = project_id,
+      dataset_name = dataset_name,
+      storage_location = output_directory,
+      datatype = "metadata",
+      max_table_id = GetMaxTableId.max_table_id,
+      schema = metadata_schema,
+      superpartitioned = "false",
+      partitioned = "false",
+      uuid = "",
```

Comment:
@kcibul @ahaessly should we include uuid as an optional input to the WDL? or rip this option (to prepend tables with a uuid) out altogether? this is currently just hardcoded to nothing.

Reply:
I haven't used the UUID piece before, I think it was from earlier testing but now I would just create a new dataset instead of tables with a prefix.  Remove it?  (@ahaessly wdyt?)

Reply:
This was definitely used for automated integration testing. I think Megan added it. If we wanted to add a uuid to the dataset, I think we would need to create that dataset outside of this wdl. But we should be able to do that in the test itself. Assuming we are not running that integration test, I would say let's go ahead and remove it.

Reply:
ok let's keep it for now until we decide what we're doing with integration testing.